snapcraft/internal/lifecycle.py

```diff
@@ -326,8 +326,12 @@ def _tar_filter(tarinfo):
 
 def containerbuild(step, project_options, output=None, args=[]):
     config = snapcraft.internal.load_config(project_options)
+    remote = os.environ.get('SNAPCRAFT_CONTAINER_BUILDS')
+    if remote in ['1', '0', None, '']:
+        remote = None
```

Comment:
You're misunderstanding the code. This is so you can do SNAPCRAFT_CONTAINER_BUILDS=myremote to specify a remote or SNAPCRAFT_CONTAINER_BUILDS=1 to use the default (local) remote. There's no such thing is an invalid value here.